### ISSUE TITLE
[Grid] stackable grid has extra left/right margin when stacked

### ISSUE THREAD

**Issue by kevinresol** [fomantic/Fomantic-UI#1907]
# Bug Report

Stackable grid has extra left/right margin when stacked. See image below how the left margin of column 1 becomes wider when the grid is stacked.

## Steps to reproduce

```html
<div class="ui basic segment">
  <div class="ui two column stackable grid">
    <div class="column">
      <div class="ui segment">
        Column 1
      </div>
    </div>
    <div class="column">
      <div class="ui segment">
        Column 2
      </div>
    </div>
  </div>
</div>
```

## Expected result
No extra margin when stacked. i.e. same amount of margin in both stacked and unstacked state

## Actual result
Stackable grid has extra left/right margin when stacked.

## Testcase
https://jsfiddle.net/04rj9sx2/2/

## Screenshot (if possible)

![Screenshot 2021-03-05 at 11 42 40 AM](https://user-images.githubusercontent.com/1103788/110064391-fb0ee580-7da7-11eb-85fe-6d598fa3d882.png)
![Screenshot 2021-03-05 at 11 42 49 AM](https://user-images.githubusercontent.com/1103788/110064394-fcd8a900-7da7-11eb-808c-1e1f2b51c7c2.png)

## Version

2.8.7

**Comment by hugopeek** [comment]
A basic segment still has padding: https://fomantic-ui.com/elements/segment.html#basic

So you'd have to set that to 0 manually, or use a different container..

**Comment by lubber-de** [comment]
I am unsure if we should completely remove the additional gutter on mobile as stated in #2269 (it seems to be a design choice of SUI, perhaps to make sure fingers holding the mobile phone won't cover content'

However, by the mentioned PR it's now possible to customize the gutter value.
If you set this to `-1rem` it solves our issue
See https://jsfiddle.net/lubber/ybjn5vgs/2/

**Comment by lubber-de** [comment]
Other workarounds:

- Use `container` instead of `segment` (as hugopeek  mentioned)
- Use `stackable page grid` instead of `stackable grid`

**Comment by lubber-de** [comment]
Fixed by #2269 
See your adjusted jsfiddle here https://jsfiddle.net/lubber/vqj0xdg1/11/
